.github/workflows/dart.yml

```diff
@@ -0,0 +1,25 @@
+name: Dart
+
+on:
+  push:
+    branches: [main]
+  pull_request:
+    branches: [main]
+
+jobs:
+  test:
+    runs-on: ${{ matrix.os }}
+    strategy:
+      matrix:
+        os: [ubuntu-latest, macos-latest, windows-latest]
+        channel: [stable, beta, dev]
```

Comment:
edge?

Reply:
Adding bleeding edge is being discussed in #4.